pkg/innerring/processors/netmap/process_peers.go

```diff
@@ -47,6 +51,28 @@ func (np *Processor) processAddPeer(ev netmapEvent.AddPeer) {
 		return
 	}
 
+	var info clientcore.NodeInfo
+	err = clientcore.NodeInfoFromRawNetmapElement(&info, netmapCore.Node(nodeInfo))
+	if err != nil {
+		np.log.Warn("could not convert node info: %w", zap.Error(err))
+		return
+	}
+
+	c, err := np.clientSource.Get(info)
+	if err != nil {
+		np.log.Warn("could not create client to the node: %w",
+			zap.Error(err),
+			zap.String("node", hex.EncodeToString(info.PublicKey())))
+		return
+	}
+
+	_, err = c.EndpointInfo(context.Background(), client.PrmEndpointInfo{})
+	if err != nil {
+		np.log.Warn("could not ping node with `EndpointInfo`",
+			zap.Error(err),
+			zap.String("node", hex.EncodeToString(info.PublicKey())))
+	}
+
```

Comment:
Close the connection immediately afterwards?

Reply:
the client was received from the [client cache](https://github.com/nspcc-dev/neofs-node/blob/9224e9f2038d1932027a5c4b01032ed904a93dcc/pkg/innerring/rpc.go#L46-L55) so it is not easy to close it cause there is some complex logic that may break the IR if a client is closed outside

Reply:
We can bypass client cache since it's just a probe, open a connection manually and close it.

Reply:
dont mind dropping the cache